industry_restaurant/data/pos_config.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_config_main_seven_star_restaurant" model="pos.config">
+        <field name="name">Seven Star Restuarant - Sans Francisco</field>
+        <field name="use_pricelist" eval="1"/>
+        <field name="ship_later" eval="1"/>
+        <field name="cash_rounding" eval="1"/>
+        <field name="start_category" eval="1"/>
+        <field name="iface_customer_facing_display_local" eval="1"/>
+        <field name="iface_splitbill" eval="1"/>
+        <field name="iface_printbill" eval="1"/>
+        <field name="iface_tipproduct" eval="1"/>
+        <field name="iface_big_scrollbars" eval="1"/>
+        <field name="iface_start_categ_id" ref="pos_category_2"/>
+        <field name="gift_card_settings">scan_use</field>
+        <field name="is_header_or_footer" eval="1"/>
+        <field name="receipt_footer">Thank You! Visit Again.</field>
+        <field name="module_pos_restaurant_appointment" eval="1"/>
+        <field name="appointment_type_ids" eval="[(6, 0, [ref('appointment_type_1'), ref('appointment_type_2')])]"/>
+        <field name="rounding_method" ref="account_cash_rounding_1"/>
+        <field name="floor_ids" eval="[(6, 0, [ref('restaurant_floor_1'), ref('restaurant_floor_2')])]"/>
+        <field name="payment_method_ids" eval="[
+            (6, 0, [
+                ref('pos_payment_method_1'),
+                ref('pos_payment_method_2'),
+            ])
+        ]"/>
+    </record>
+    <record id="pos_restaurant.pos_config_main_restaurant" model="pos.config">
+        <field name="active" eval="False"/>
+    </record>
+
+    <function name="add_cash_payment_method" model="pos.config"/>
```

Comment:
I don't see the value of this.

Reply:
So the consultant has adapted the new cash payment method for this particular POS so as per that I have done this

Reply:
But if I understand correctly, it only edits `payment_method_ids`. Why couldn't you do it before?